source/isaaclab_mimic/isaaclab_mimic/motion_planners/curobo/curobo_planner_config.py

```diff
@@ -0,0 +1,459 @@
+# Copyright (c) 2024-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import tempfile
+import yaml
+from dataclasses import dataclass, field
+
+from curobo.geom.sdf.world import CollisionCheckerType
+from curobo.geom.types import WorldConfig
+from curobo.util_file import get_robot_configs_path, get_world_configs_path, join_path, load_yaml
+
+from isaaclab.utils.assets import ISAACLAB_NUCLEUS_DIR, retrieve_file_path
+
+
+@dataclass
+class CuroboPlannerConfig:
```

Comment:
Suggest renaming `Config` to `Cfg` to stay consistent with rest of our codebase. Same goes for the file name.